File: VB/CustomGridView/CustomGridViewDesigner.vb
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.ComponentModel
Imports System.Windows.Forms
Imports DevExpress.Web.ASPxGridView.Design
Imports DevExpress.Web.ASPxGridView

Namespace CustomGridView
    <ToolboxItem(False)> _
    Public Class CustomGridViewDesigner
        Inherits GridViewDesigner

        Public Overrides Sub Initialize(ByVal component As IComponent)
            MyBase.Initialize(component)
            RegisterTagPrefix(GetType(ASPxGridView))
        End Sub
    End Class
End Namespace

File: CS/CustomGridView/CustomGridViewDesigner.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel;
using System.Windows.Forms;
using DevExpress.Web.ASPxGridView.Design;
using DevExpress.Web.ASPxGridView;

namespace CustomGridView {
    [ToolboxItem(false)]
    public class CustomGridViewDesigner : GridViewDesigner {
        public override void Initialize(IComponent component) {
            base.Initialize(component);
            RegisterTagPrefix(typeof(ASPxGridView));
        }
    }
}
